src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -1498,30 +1499,34 @@ public static function providePadStart()
     /**
      * @dataProvider provideTruncate
      */
-    public function testTruncate(string $expected, string $origin, int $length, string $ellipsis, bool $cut = true)
+    public function testTruncate(string $expected, string $origin, int $length, string $ellipsis, TruncateCut $cut = TruncateCut::Char)
```

Comment:
should we keep boolean test cases? the signature allows `bool|TruncateCut`

Reply:
I forgot to add them back after removing the deprecation, thanks !